tests/lit-tests/vectorcall-1.ispc

```diff
@@ -0,0 +1,27 @@
+// RUN: %{ispc}  %s --emit-asm --target=avx2-i32x8 -o - | FileCheck %s -check-prefix=CHECK_AVX2
```

Comment:
Comment describing what you are checking here is needed. I.e. "Check that calling convention is not adding any overhead and function is optimized to only add and ret instructions"